domain/src/main/java/starter/project/domain/Engine.java

```diff
@@ -12,5 +12,5 @@
 @NoArgsConstructor
 @AllArgsConstructor
 public class Engine {
-    private int cylinders;
+    private int engine;
```

Comment:
What does this do?

Reply:
it was just a car thing.I remembered that not every care uses cylinders. So the naming was just abit confusing